variables.tf

```diff
@@ -175,6 +175,16 @@ variable "node_pools_tags" {
   }
 }
 
+variable "node_pools_oauth_scopes" {
+  type        = "map"
+  description = "Map of lists containing node oauth scopes by node-pool name"
+
+  default = {
+    all               = ["https://www.googleapis.com/auth/cloud-platform"]
```

Comment:
`cloud-platform` is fine as a default and recommended.

IAM should be preferred for access control over scopes.

Reply:
Thanks for the review,  I wass pulling my hair out about the CI because I couldn't see the errors and their cause.

For the initial comment: The doc referenced only says that GKE no more adds default scopes (such as compute-rw) since Kubernetes 1.9, they only add really necessary internal scope for security issues (the major reason for this PR).

So, I believe that this PR is compatible with any Kubernetes and GKE version. It doesn't change the default behavior of the module (who puts cloud-platform as hardcoded), just add some customized option for new pools.

Reply:
@lantier gotcha, thanks for providing this context!